src/hotspot/share/memory/metaspace/virtualSpaceNode.cpp

```diff
@@ -109,7 +109,7 @@ bool VirtualSpaceNode::commit_range(MetaWord* p, size_t word_size) {
   }
 
   // Commit...
-  if (os::commit_memory((char*)p, word_size * BytesPerWord, false) == false) {
+  if (os::commit_memory((char*)p, word_size * BytesPerWord, !ExecMem, mtMetaspace) == false) {
```

Comment:
Not sure if I suggested something different in my first review, but thinking this over, please don't hard wire mtMetaspace, take the flag from the ReservedSpace member of VirtualSpaceNode.

The reason is that metaspace can be used for at least two different flags, and may later be expanded for more.

Reply:
Done.

Reply:
Where? I still see mtMetaspace.

Reply:
It should be seen now.